pymongoexplain/commands.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pymongo
+from bson.son import SON
+from typing import Union
```

Comment:
Let's follow a similar convention for imports as we do in Pymongo:

* Standard library imports come first in alphabetical order.
* Dependency package imports come next in alphabetical order.
* Imports from within the same module come last in alphabetical order.

Leave a blank line after each category of imports.